### PR TITLE
#487 Test ping after pr completed does not reassign

### PR DESCRIPTION
**Pull request by rgevaerd**
#487 
Checking the footprint provided [here](https://github.com/zerocracy/farm/issues/487#issuecomment-401525930), the problem would be a `Start order` issued after the pr finished and points awarded. Searching current stalkholders, the one that would issue such claim with the attributes in the footprint would be `assign_performer.groovy`. As this stakeholder consumes `Ping` claims, I am creating a test that has a ping claim after the pr completed.

### CODE REVIEW

**Comment by krzyk** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1531#issuecomment-410015437))

@krzyk This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1531](https://github.com/zerocracy/farm/issues/1531) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231531%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zerocracy/farm/issues/1531#issuecomment-410285475))

@g4s8 Job [#1531](https://github.com/zerocracy/farm/issues/1531) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1531](https://github.com/zerocracy/farm/issues/1531) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2151623774); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.zerocracy.bundles.close_job_with_quality_review
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy]: @rgevaerd wrong package name

File: src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_before.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.zerocracy.bundles.close_job_with_quality_review
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_before.groovy]: @rgevaerd package name should end with `ping_after_pr_completed`

File: src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+import com.zerocracy.pm.in.Orders
+import com.zerocracy.pm.scope.Wbs
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  Wbs wbs = new Wbs(project).bootstrap()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy]: @rgevaerd you can inline `wbs`

File: src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  MatcherAssert.assertThat(wbs.iterate(), Matchers.empty())
+  Orders orders = new Orders(project).bootstrap()
+  MatcherAssert.assertThat(orders.iterate(), Matchers.empty())
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_after.groovy]: @rgevaerd you can inline `orders`

File: src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_before.groovy [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.jcabi.github.*
+import com.jcabi.xml.XML
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.entry.ExtGithub
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  Github github = new ExtGithub(farm).value()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/ping_after_pr_completed/_before.groovy]: @rgevaerd `github` can be inlined

**Comment by krzyk** [comment]
@rgevaerd thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by krzyk** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@rgevaerd so you want to say that this bug (#487) is not reproducing anymore?

**Comment by rgevaerd** [comment]
@g4s8 Based on [the logic I implemented](https://github.com/zerocracy/farm/pull/1531#issue-205619460), yes, it is not reproducing. The issue reporter also expected it to be no longer reproducing, as he commented at https://github.com/zerocracy/farm/issues/487#issuecomment-401367843

**Comment by g4s8** [comment]
@rgevaerd got it, thanks

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1531](https://github.com/zerocracy/farm/issues/1531) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**